packages/vue-docgen-api/src/script-handlers/__tests__/slotHandlerLitteral.ts

```diff
@@ -118,4 +118,22 @@ export default {
 		}
 		expect(mockSlotDescriptor.bindings).toMatchObject([{ name: 'index' }, { name: 'content' }])
 	})
+
+	it('should allow for slots defined by a setup function', async () => {
+		const src = `
+export default {
+  /**
+   * @slot icon
```

Comment:
I would like to have a test to document (and maintain) that bindings are parsed if you don't mind.
Otherwise the PR looks ♨️ 

```suggestion
   * @slot icon
   * @binding {number} size - size asked for the icon
```